pkg/platform/aws/aws_test.go

```diff
@@ -0,0 +1,53 @@
+// Copyright 2020 The Lokomotive Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package aws //nolint:testpackage
+
+import "testing"
+
+func TestCheckSamePortForMuliplePools(t *testing.T) {
+	c := config{
+		WorkerPools: []workerPool{
+			{
+				Name: "pool-1",
+			},
+			{
+				Name: "pool-2",
+			},
+		},
+	}
+
+	if d := c.checkDifferentPortForMuliplePools(); !d.HasErrors() {
+		t.Error("Should fail with lb_http_port and lb_https_port not set")
+	}
+}
+
+func TestCheckDifferentPortForMuliplePools(t *testing.T) {
+	c := config{
+		WorkerPools: []workerPool{
+			{
+				Name: "pool-1",
+			},
+			{
+				Name:        "pool-2",
+				LBHTTPPort:  8080,
+				LBHTTPSPort: 8443,
```

Comment:
If you set these to 80/443 respectively, tests will pass, but cluster provisioning will still fail.